l10n_it_account_balance_report/__manifest__.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Openforce (http://www.openforce.it)
+# Copyright 2019 Alessandro Camilli (alessandrocamilli@openforce.it)
+# Copyright 2019 Silvio Gregorini (silviogregorini@openforce.it)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "Italian Localisation -"
+            " Stampe Stato Patrimoniale e Conto Economico",
+    'summary': "Report PDF e XLS per Stato Patrimoniale e Conto Economico"
+               " a sezioni contrapposte",
```

Comment:
Come da [linee guida](https://www.odoo-italia.org/documentazione/12.0/sviluppo/guidelines.html#titolo-del-modulo) direi `ITA -`, inoltre stato patrimoniale e conto economico andrebbero con la lettera minuscola.

P.S.: in questo caso, per uniformità con le attuali traduzioni di contabilità (e con il [glossario](https://www.odoo-italia.org/documentazione/12.0/traduzioni/glossari/contabilita.html)) sarebbe consigliabile tradurre `Report` con `Rendiconto`.

Reply:
> Come da [linee guida](https://www.odoo-italia.org/documentazione/12.0/sviluppo/guidelines.html#titolo-del-modulo) direi `ITA -`

Scusa, probabilmente mi sono espresso male.
In `name` bastava sostituire `Italian Localization -` con `ITA -` senza aggiungere `Rendicontazione`.
Se invece è una cosa voluta no problem. ;-)

P.S.: il `report` a cui mi riferivo è quello nel `summary`, per uniformità sarebbe da sostituire con `Rendiconto` oppure `Rendiconti`.


>  inoltre stato patrimoniale e conto economico andrebbero con la lettera minuscola.

Sono presenti anche in `summary`.